src/astro/databases/base.py

```diff
@@ -41,6 +41,10 @@ def hook(self) -> BaseHook:
         """Return an instance of the database-specific Airflow hook."""
         raise NotImplementedError
 
+    def replace_illegal_columns_chars(self, char: str) -> str:
```

Comment:
Perhaps, from a Database perspective, we could just define the following attributes:
* `illegal_characters: List[chr]`  
* `illegal_characters_replacement: chr: `
And allow the operator to use them as needed.

Reply:
Done.